rust/src/web/graphql/transactions/mod.rs

```diff
@@ -59,6 +60,20 @@ impl TransactionsQueryRoot {
             }
         };
 
+        // block height query
+        if let Some(block_height) = query.block_height {
+            let mut transactions: Vec<TransactionWithBlock> = db
+                .get_blocks_at_height(block_height as u32)?
```

Comment:
Should we (somehow) support providing a limit when querying the database?

I suppose we can't do that due to sorting.

Reply:
This won't ever be an expensive call. I've only ever seen 2-3 blocks per block height and there are a max of 128 txns per block.